pkg/host/internal/network/network.go

```diff
@@ -329,3 +361,23 @@ func (n *network) EnableHwTcOffload(ifaceName string) error {
 	log.Log.V(0).Info("EnableHwTcOffload(): feature is still disabled, not supported by device", "device", ifaceName)
 	return nil
 }
+
+// GetNetDevLinkAdminState returns the admin state of the interface.
+func (n *network) GetNetDevLinkAdminState(ifaceName string) string {
+	log.Log.V(2).Info("GetNetDevLinkAdminState(): get LinkAdminState", "device", ifaceName)
+	if len(ifaceName) == 0 {
+		return ""
+	}
+
+	link, err := n.netlinkLib.LinkByName(ifaceName)
+	if err != nil {
+		log.Log.Error(err, "GetNetDevLinkAdminState(): failed to get link", "device", ifaceName)
+		return ""
+	}
+
+	if link.Attrs().Flags&net.FlagUp == 0 {
+		return "down"
```

Comment:
please put this one as a const and use it also in https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/587/files#diff-371d95e79c61d8cc1e6791d27e009289552efcaada38f9b2fc75ebb487c90ea7R275